### DISCUSSION TITLE
LLDPCLI difference between requested/allocated wattage

### DISCUSSION THREAD

**Discussion by bryceremick** [lldpd/lldpd#552] [Q&A] [answered]
Hi, 

I am somewhat new to LLDP, and I am confused by the `lldpcli configure dot3 power pd` command. 

What is the difference between the `requested` and `allocated` arguments in this command? I'd be very appreciative if someone could explain what these arguments are for.

A little bit of context: I'm implementing `lldp` for an android device that needs `45` watts of power.

Thanks!

**Comment by vincentbernat** [comment, marked as answer]
The requested is what the "powered device" is asking for. The allocated is what the "power sourcing equipment" is providing. Notably, it can provides less than what is requested. In your case, you are a PD device, you should use "requested".

**Reply by bryceremick** [reply]
Thank you for resolving that for me! I really appreciate it.

One more question I have:

What is the best way to verify that the PSE has allocated the desired amount of power? Is this done using the `show neighbors` command? Or the `show interfaces` command... ?

Thanks,

**Reply by vincentbernat** [reply]
`show neighbors`
